src/components/controls.vue

```diff
@@ -154,9 +193,37 @@ export default {
     },
 
     onAddOverlay: function () {
-      alert('adding overlays in this demo is not implemented yet! :)')
+      if(this.addOverlyEnabled) {
+        this.dialog = true;
+      }
+      else {
+        alert('adding overlays in this demo is not implemented yet! :)')
+      }
+    },
+
+
+    visibilityIcon: function(val) {
```

Comment:
This is a duplicate function (there are two methods named `visibilityIcon`. I deleted one locally to get the demo server working, but it would be good to remove it here.

Reply:
at least in my `npm run serve` console I get the `duplicate key` error. Did you have any errors?